quickstarts/library-process/README.md

```diff
@@ -78,186 +57,114 @@ $ oc new-app example-app-secret -p SECRET_NAME=businesscentral-app-secret
 Before proceed, make sure you have the RHDM imagestreams available under the 'openshift' namespace.
 
 ```bash
-$ oc get imagestream rhpam-kieserver-rhel8 -n openshift | grep 7.8
+$ oc get imagestream rhpam-kieserver-rhel8 -n openshift | grep 7.8.1
 Error from server (NotFound): imagestreams.image.openshift.io "rhpam-kieserver-rhel8" not found
 
 ```
  If the `rhpam-kieserver-rhel8` is not found, install it under the 'openshift' namespace:
  ```bash
-$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/master/rhpam78-image-streams.yaml -n openshift
+$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/7.8.1.GA/rhpam78-image-streams.yaml -n openshift
```

Comment:
There should be placed release branch instead of tag (and to other places as well), also this link does not work as tag does not exists yet.
```suggestion
$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhpam-7-openshift-image/7.8.x/rhpam78-image-streams.yaml -n openshift
```

Reply:
make sense.